openmdao/drivers/tests/test_scipy_optimizer.py

```diff
@@ -1098,9 +1096,7 @@ def test_feature_optimizer(self):
         model.add_subsystem('p2', IndepVarComp('y', 50.0), promotes=['*'])
         model.add_subsystem('comp', Paraboloid(), promotes=['*'])
 
-        prob.driver = ScipyOptimizeDriver()
-        prob.driver.options['optimizer'] = 'COBYLA'
-        prob.driver.options['disp'] = True
+        prob.driver = ScipyOptimizeDriver(optimizer='COBYLA')
```

Comment:
This looks like a feature test, so I think the intent was to keep "disp" set to True here.

Reply:
True is the default, and the example was only meant to illustrate setting the optmizer, per the text.

Reply:
(so no reason to reinforce the already-default setting)